### PR TITLE
CCM-1089: PR Bot Fix

### PR DESCRIPTION
**Pull request by ChewieCB**
Updated the bot regex to search for CCM prefixed strings

### CODE REVIEW

File: .github/workflows/pr-lint.yaml [hunk 1]
```diff
@@ -5,18 +5,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check ticket name conforms to requirements
-        run: echo ${{ github.event.pull_request.head.ref }} | grep -i -E -q "(apm-[0-9]+)|(apmspii-[0-9]+)|(adz-[0-9]+)|(amb-[0-9]+)|(dependabot\/)"
+        run: echo ${{ github.event.pull_request.head.ref }} | grep -i -E -q "(ccm-[0-9]+)|(apmspii-[0-9]+)|(adz-[0-9]+)|(ccm-[0-9]+)|(dependabot\/)"
```
Comment (PhilipSkinner) [inline .github/workflows/pr-lint.yaml:8]: can simplify this to remove the apmspii, adz and duplicated ccm regexes

File: .github/workflows/pr-lint.yaml [hunk 2]
```diff
@@ -5,18 +5,18 @@ jobs:
...
 
       - name: Grab ticket name
-        if: contains(github.event.pull_request.head.ref, 'apm-') || contains(github.event.pull_request.head.ref, 'APM-') || contains(github.event.pull_request.head.ref, 'apmspii-') || contains(github.event.pull_request.head.ref, 'APMSPII-') || contains(github.event.pull_request.head.ref, 'adz-') || contains(github.event.pull_request.head.ref, 'ADZ-') || contains(github.event.pull_request.head.ref, 'amb-') || contains(github.event.pull_request.head.ref, 'AMB-')
-        run: echo ::set-env name=TICKET_NAME::$(echo ${{ github.event.pull_request.head.ref }} | grep -i -o '\(apm-[0-9]\+\)\|\(apmspii-[0-9]\+\)\|\(adz-[0-9]\+\)|\(amb-[0-9]\+\)' | tr '[:lower:]' '[:upper:]')
+        if: contains(github.event.pull_request.head.ref, 'ccm-') || contains(github.event.pull_request.head.ref, 'CCM-') || contains(github.event.pull_request.head.ref, 'apmspii-') || contains(github.event.pull_request.head.ref, 'APMSPII-') || contains(github.event.pull_request.head.ref, 'adz-') || contains(github.event.pull_request.head.ref, 'ADZ-') || contains(github.event.pull_request.head.ref, 'ccm-') || contains(github.event.pull_request.head.ref, 'CCM-')
```
Comment (PhilipSkinner) [inline .github/workflows/pr-lint.yaml:12]: same here, remove the irrelevant checks for apmspii, adz and the duplicate CCM checks

File: .github/workflows/pr-lint.yaml [hunk 3]
```diff
@@ -5,18 +5,18 @@ jobs:
...
         continue-on-error: true
         env:
           ACTIONS_ALLOW_UNSECURE_COMMANDS: true
 
       - name: Comment on PR with link to JIRA ticket
-        if: contains(github.event.pull_request.head.ref, 'apm-') || contains(github.event.pull_request.head.ref, 'APM-') || contains(github.event.pull_request.head.ref, 'apmspii-') || contains(github.event.pull_request.head.ref, 'APMSPII-') || contains(github.event.pull_request.head.ref, 'adz-') || contains(github.event.pull_request.head.ref, 'ADZ-') || contains(github.event.pull_request.head.ref, 'amb-') || contains(github.event.pull_request.head.ref, 'AMB-')
+        if: contains(github.event.pull_request.head.ref, 'ccm-') || contains(github.event.pull_request.head.ref, 'CCM-') || contains(github.event.pull_request.head.ref, 'apmspii-') || contains(github.event.pull_request.head.ref, 'APMSPII-') || contains(github.event.pull_request.head.ref, 'adz-') || contains(github.event.pull_request.head.ref, 'ADZ-') || contains(github.event.pull_request.head.ref, 'ccm-') || contains(github.event.pull_request.head.ref, 'ccm-')
```
Comment (PhilipSkinner) [inline .github/workflows/pr-lint.yaml:19]: same here, remove the irrelevant checks for apmspii, adz and the duplicate ccm checks

**Review by PhilipSkinner — CHANGES_REQUESTED** [review]

**Review by Kennedy1265 — APPROVED** [review]

**Pull request merged**